torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -145,16 +146,30 @@ void ProfilingRecord::insertShapeProfile(Node* n, Value* i) {
   n->replaceInputWith(i, pn->output());
 }
 
+bool isProfilable(Node* n) {
+  if (isSupported(n)) {
```

Comment:
Nit: maybe put this function into `tensorexpr::` namespace or give it a less generic name? It's kind of hard to understand what `isSupported` in this context means unless you already know it.

Reply:
done